runelite-client/src/main/java/net/runelite/client/plugins/fishing/FishingOverlay.java

```diff
@@ -54,6 +56,28 @@
 
 	private final PanelComponent panelComponent = new PanelComponent();
 
+	private final ImmutableSet<Integer> fishingAnimations = ImmutableSet.of(
```

Comment:
This field can be made `static`, be defined as a `Set` (or even a `Collection` since we're not calling any set-specific methods of it), and renamed to match the constant naming schema:
```suggestion
	private static final Set<Integer> FISHING_ANIMATIONS = ImmutableSet.of(
```